src/binary/Y2RubyComponent.cc

```diff
@@ -75,16 +76,30 @@ Y2Namespace *Y2RubyComponent::import (const char* name)
   args->add (YCPString(/*module*/ name));
   args->add (YCPString(/*module*/ module));
 
-  try {
-    YRuby::loadModule (args);
+  if (YRuby::loadModule (args))
+  {
     y2debug("Module '%s' loaded", name);
     // introspect, create data structures for the interpreter
     Y2Namespace * res = new YRubyNamespace (name);
     namespaces[name] = res;
     return res;
-  } catch (exception& e) {
```

Comment:
1 and 2 is correct, but YRubyNamespace does throw:
- https://github.com/yast/yast-ruby-bindings/blob/e41a0a1385c2ad6183ee6d903d59e5d148093205/src/binary/YRubyNamespace.cc#L312
- https://github.com/yast/yast-ruby-bindings/blob/e41a0a1385c2ad6183ee6d903d59e5d148093205/src/binary/YRubyNamespace.cc#L339